IDE/MonoDevelop/MonoDevelop.MonoGame/templates/Common/Program.cs

```diff
@@ -13,69 +13,69 @@
 
 namespace ${Namespace}
 {
-	#if __IOS__ || __TVOS__
-	[Register("AppDelegate")]
-	class Program : UIApplicationDelegate
-	#else
-	static class Program
-	#endif
+    #if __IOS__ || __TVOS__
+    [Register("AppDelegate")]
+    class Program : UIApplicationDelegate
+    #else
+    static class Program
+    #endif
     {
-        private static Game1 game;
-
-		internal static void RunGame() 
-		{
-			game = new Game1();
-			game.Run();
-		}
-			
+        internal static void RunGame() 
+        {
+            using (var game = new Game1())
+            {
+                game.Run();
```

Comment:
We need separate methods for Desktop/iOS, on desktop we should be doing the above to get proper Disposing.

On the side note, I think the whole Program.cs file is a mess, and that we should be using perplatform Program.cs instead of one filled with preprocessing params... just saying...